src/commands/new.ts

```diff
@@ -828,11 +836,7 @@ module.exports = {
 
       // #region Remove Demo code
       const removeDemoPart = removeDemo === true ? "code" : "markup"
-      startSpinner(`Removing fancy demo ${removeDemoPart}`)
-
-      p(yellow(`removeDemo: ${removeDemo}`))
-      p(yellow(`removeDemoPart: ${removeDemoPart}`))
```

Comment:
I assumed these `p()` calls were incorrectly committed in #2949

Reply:
Yeah looks like it, thanks